src/engine/order.ts

```diff
@@ -137,8 +136,8 @@ export class OrderEngine {
 
     // Order slipped
```

Comment:
i think here we should add a accepted slipped-%. Otherwise you will always create and delete orders just because of a few cents. I am not sure if thats the plan.